project-2-e-commerce/src/component/home/NewProducts.js

```diff
@@ -0,0 +1,58 @@
+import Carousel from "react-multi-carousel"
+import "react-multi-carousel/lib/styles.css"
+import { useDispatch, useSelector } from 'react-redux';
+import { useEffect } from 'react';
+import ProductsItem from '../product-item';
+import { responsive } from './helpers'
+import { getNewProds } from '../../store/slices/ProductsSlice'
+import LoadingComponent from '../loading/index'
+import { useTranslation } from 'react-i18next'
+
+function NewProducts() {
+    const dispatch = useDispatch()
+    const newList = useSelector(state => state.products.new)
+    const err = useSelector(state => state.products.error)
```

Comment:
```suggestion
    const { new: newList, error: err } = useSelector(({products}) => products)
```
Thử destructuring ngắn gọn như này xem sao anh nhé